cartoframes/datasets.py

```diff
@@ -115,7 +116,7 @@ def _copyfrom(self, with_lonlat=None):
         self.cc.copy_client.copyfrom(
             """COPY {table_name}({columns},the_geom)
                FROM stdin WITH (FORMAT csv, DELIMITER '|');""".format(table_name=self.table_name, columns=columns),
-            self._rows(self.df, self.df.columns, with_lonlat, geom_col)
+            self._rows(self.df, [c for c in self.df.columns if c != 'cartodb_id'], with_lonlat, geom_col)
```

Comment:
Probably doesn't matter too much, but I wonder if a set difference would be faster?
```python
set(self.df.columns) - {'cartodb_id', }
```

Reply:
Good point, I've been looking at blogposts and it may be more efficient than the list comprehension thing. The fact is that we need to preserve the order to generate the rows with the same column order than in the header.